# This represents a vector in free space.

# This is semantically different than a point.
# A vector is always anchored at the origin.
# When a transform is applied to a vector, only the rotational component is applied.
# import out of underlay "/../../../../share/geometry_msgs/msg/Point.msg"

float64 x
float64 y
float64 z